### PR TITLE
Manage replicator task

### PR DESCRIPTION
**Pull request by lmmrssa**

### CODE REVIEW

**Comment by lmmrssa** [comment]
On registration create replicator to get status of request
On manager sync will push all db to parent
Use single sync service with course resource pull as well
Change replicator from continuous to one time

File: src/app/shared/sync.service.ts [hunk 1]
```diff
@@ -64,24 +64,9 @@ export class SyncService {
   private syncParams(opt, credentials, type) {
     const dbSource = opt.dbSource || opt.db;
     const dbTarget = opt.dbTarget || opt.db;
-    const parentDomain = opt.parentDomain || this.userService.getConfig().parentDomain;
-    const code = opt.code || this.userService.getConfig().code;
```
Comment (lmmrssa) [inline src/app/shared/sync.service.ts]: we will not have userService when we are doing configuration so we do not have code as well. That's why had kept code on opt.

Comment (paulbert) [reply src/app/shared/sync.service.ts]: See line 40.  It's included there.

Comment (lmmrssa) [reply src/app/shared/sync.service.ts]: sorry @paulbert I'm pointing to code you use on line 95. you need opt for code as well.

File: src/app/shared/sync.service.ts [hunk 2]
```diff
@@ -23,76 +22,85 @@ const passwordFormFields = [
...
-    private planetMessageService: PlanetMessageService
+    private dialogsFormService: DialogsFormService
   ) {}
 
-  private defaultConfig = {
-    'create_target':  false,
-    'continuous': false,
-    'owner': this.userService.get().name
-  };
+  confirmPasswordAndRunReplicators(replicators) {
```
Comment (lmmrssa) [inline src/app/shared/sync.service.ts]: there will be cases when we do not need to ask for password confirmation. Such as when we are doing setup of application. So we will require replicator to be able to run without password confirmation

**Pull request merged**